components/Dialogs/src/MDCAlertController.m

```diff
@@ -202,6 +226,15 @@ - (void)setMessage:(NSString *)message {
   }
 }
 
+- (void)setAttributedMessage:(NSAttributedString *)attributedMessage {
+  _attributedMessage = [attributedMessage copy];
+  if (self.alertView) {
+    self.alertView.messageLabel.attributedText = attributedMessage;
+    self.preferredContentSize =
+        [self.alertView calculatePreferredContentSizeForBounds:CGRectInfinite.size];
+  }
+}
```

Comment:
Aye sorry for the rounds of iterative feedback here, but I noticed here that we need to make the attributed text assignment logic consistent across all setters because the displayed behavior is now dependent on both the message and attributedMessage properties.

With this in mind, my recommendation is to refactor the following logic out from setupAlertView into a method:

```objc
- (void)messageDidChange {
  if (self.attributedMessage.length > 0) {
    self.alertView.messageLabel.attributedText = self.attributedMessage;
  } else {
    self.alertView.messageLabel.text = self.message;
  }
  self.preferredContentSize =
      [self.alertView calculatePreferredContentSizeForBounds:CGRectInfinite.size];
}
```

And then to call this method every time the message or attributedMessage changes and the alert view is loaded. That method will also need to be invoked from setupAlertView.

```objc
- (void)setAttributedMessage:(NSAttributedString *)attributedMessage {
  _attributedMessage = [attributedMessage copy];
  if (self.alertView) {
    [self messageDidChange];
  }
}
```

Reply:
> Aye sorry for the rounds of iterative feedback here, but I noticed here that we need to make the attributed text assignment logic consistent across all setters because the displayed behavior is now dependent on both the message and attributedMessage properties.

No worries! Part of that was on me since I didn't build the project up until recently when I managed to configure it!

Just pushed the changes 😄